scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py

```diff
@@ -131,6 +134,17 @@ def create_extract_samples_table(control_samples, fq_destination_table_samples,
     return query_return['results']
 
 
+def get_location_filters_from_interval_list(interval_list):
+    interval_test = pybedtools.BedTool(interval_list)
+    location_clause_list = [f"""(location >= {CHROM_MAP[interval.chrom]}{'0' * (12 - len(str(interval.start)))}{interval.start} 
+            AND location <= {CHROM_MAP[interval.chrom]}{'0' * (12 - len(str(interval.end)))}{interval.end})"""
+                            for interval in interval_test]
+    if len(location_clause_list) > 5000:
+        print(f"\n\nTrying to query over the limit of 5,000 locations; {interval_list} will be discarded, and all locations will be queried.\n\n")
+        return ""
```

Comment:
Shouldn't this test happen before the potentially expensive forcomp starting on line 139 (substituting `len(interval_test)` for `len(location_clause_list)`?